spec/classes/foreman_proxy__config__spec.rb

```diff
@@ -21,6 +28,7 @@
         usr_dir = '/usr'
         var_dir = '/var/lib/puppet'
       end
+      puppet_etc_dir = facts[:osfamily] == 'Archlinux' ? var_dir : "#{etc_dir}/puppet"
```

Comment:
while the result is correct, setting something with "etc_dir" in it to "var_dir" looks funny.